SWLOR.Game.Server/Feature/PerkDefinition/ForcePerkDefinition.cs

```diff
@@ -344,23 +344,21 @@ private void ForceBurst()
         }
         private void ForceMind()
         {
-            _builder.Create(PerkCategoryType.ForceLight, PerkType.ForceMind)
+            _builder.Create(PerkCategoryType.ForceUniversal, PerkType.ForceMind)
                 .Name("Force Mind")
 
                 .AddPerkLevel()
-                .Description("Converts 25% of the user's FP into HP.")
+                .Description("PLACEHOLDER.")
```

Comment:
Placeholder text here needs to be fixed.

Reply:
woops

yep